lib/rules/string-quotes/README.md

```diff
@@ -10,6 +10,8 @@ a[id="foo"] { content: "x"; }
 
 Quotes within comments are ignored.
 
+Also, the quotes in a `charset` @-rule are ignored as using single quotes in this context is incorrect according the [CSS specification](https://www.w3.org/TR/CSS2/syndata.html#x57).
```

Comment:
Let's move this line to below the _quotes in comments_ code underneath this sentence

Update the verbiage to `Single quotes in a charset @-rule are ignored as using single quotes in this context is incorrect according the CSS specification.`

Maybe see what adding a code example for this also looks like, it might look weird or add confusion 🤔